### ISSUE TITLE
Possible to add RTSP Support to Video

### ISSUE THREAD

**Issue by webdes03** [openhab/openhab#4492]
## Expected Behavior
Would be able to add RTSP streams of security cameras.

## Current Behavior
Currently doesn't support RTSP 

## Your Environment
* Version used: OpenHAB 1, Upgrading soon to 2
* Environment name and version (e.g. Chrome 39, node.js 5.4, Java 8, ...): 
* Operating System and version (desktop or mobile, Windows 10, Raspbian Jessie, ...): Raspbian

**Comment by tkuehne71** [comment]
I like this

**Comment by webdes03** [comment]
I use Ubiquiti Unifi security cameras and those streams are only available via RTSP.

**Comment by bigbasec** [comment]
I would also love to see this.

**Comment by ncrandel** [comment]
Agreed. The best way to access video feeds on most new IP cams is RTSP.

**Comment by kaikreuzer** [comment]
Does anyone have a suggestion how to handle RTSP? How to proxy it through Jetty, how to display it on HTML5, Android & iOS?

**Comment by bigbasec** [comment]
It's not going to be easy.  Even the solutions that do support it(MotionEye, zoneminder, etc) that run under linux require the use of ffmpeg or vlc to render the video.  That's actually how I do it now, my zoneminder install fetches the RTSP, and I have openHAB setup to view the stream from zoneminder, which comes to openhab as mjpeg.

**Comment by watou** [comment]
@kaikreuzer Idea: Supporting [WebRTC](https://webrtc.org/start/) from the openHAB clients ([intro video](https://www.youtube.com/watch?v=p2HzZkd2A40)), and then testing with server software like [Kurento](http://www.kurento.org/) to transcode RTSP to WebRTC.  This way no plugins are required in browser clients, and there is Android and iOS support as well.

**Comment by watou** [comment]
A very different approach is to pull [MotionEye](https://github.com/ccrisan/motioneye/wiki/Install-On-Raspbian) JPEG feeds into openHAB Image widgets.  It uses on a more recent version of motion that accepts RTSP streams.

**Comment by bigbasec** [comment]
I do exactly that, but with zone minder.  I let zone minder deal with the rtsp, and just grab the mjpeg from that to display in openhab.

**Comment by kirantpatil** [comment]
Hi all,

[Ring door bell is set up guide](https://www.youtube.com/watch?v=-gBbsayJFGI) shows that we can stream video via RTSP to the client app remotely by registering the app with the cloud registration only.

How can it be achieved using openhab-cloud registration in place ?

As @kaikreuzer mentioned, we should handle RTSP via proxy it through Jetty.

@kaikreuzer, at client we need vlc player to initiate and receive streaming via RTSP.

Thanks.
